xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-validation/xwiki-platform-model-validation-default/src/main/java/org/xwiki/model/validation/internal/EntityNameValidationConfigurationSource.java

```diff
@@ -43,11 +43,9 @@
 {
     private static final List<String> SPACE_NAMES = Arrays.asList("XWiki", "EntityNameValidation");
 
-    private static final LocalDocumentReference CLASS_REFERENCE =
-        new LocalDocumentReference(SPACE_NAMES, "ConfigurationClass");
+    static final LocalDocumentReference CLASS_REFERENCE = new LocalDocumentReference(SPACE_NAMES, "ConfigurationClass");
 
-    private static final LocalDocumentReference DOC_REFERENCE =
-        new LocalDocumentReference(SPACE_NAMES, "Configuration");
+    static final LocalDocumentReference DOC_REFERENCE = new LocalDocumentReference(SPACE_NAMES, "Configuration");
```

Comment:
Scope changed to reuse the references in `EntityNameValidationConfigurationListener`.